pkg/adapter/nimbus-kyverno/processor/kpbuilder.go

```diff
@@ -67,12 +96,76 @@ func cocoRuntimeAddition(np *v1alpha1.NimbusPolicy, rule v1alpha1.Rule) kyvernov
 	}
 	patchBytes, err := json.Marshal(patchStrategicMerge)
 	if err != nil {
-		panic(err)
+		errs = append(errs, err)
 	}
 	if err != nil {
-		panic(err)
+		errs = append(errs, err)
 	}
-	kp := kyvernov1.Policy{
+
+	deploymentsGVR := schema.GroupVersionResource{Group: "apps", Version: "v1", Resource: "deployments"}
+	deployments, err := client.Resource(deploymentsGVR).Namespace(np.Namespace).List(context.TODO(), metav1.ListOptions{})
```

Comment:
In the ListOptions, you should be able to specify the labels. https://stackoverflow.com/questions/73857627/how-to-list-pods-based-on-labels-in-kubernetes-go-client

Reply:
Does not work for list of labels, tested and verified @shivaccuknox will optimise it in another way

Reply:
I had tested the code it is not working .